terratag.go

```diff
@@ -45,24 +45,24 @@ func Terratag(args cli.Args) error {
 		return err
 	}
 
-	if err := terraform.ValidateTerraformInitRun(args.Dir); err != nil {
+	if err := terraform.ValidateInitRun(args.Dir, args.Terragrunt); err != nil {
 		return err
 	}
 
-	matches, err := terraform.GetTerraformFilePaths(args.Dir)
+	matches, err := terraform.GetFilePaths(args.Dir, args.Terragrunt)
 	if err != nil {
 		return err
 	}
 
-	counters := tagDirectoryResources(args.Dir, args.Filter, matches, args.Tags, args.IsSkipTerratagFiles, *tfVersion, args.Rename)
+	counters := tagDirectoryResources(args.Dir, args.Filter, matches, args.Tags, args.IsSkipTerratagFiles, *tfVersion, args.Rename, args.Terragrunt)
 	log.Print("[INFO] Summary:")
 	log.Print("[INFO] Tagged ", counters.taggedResources, " resource/s (out of ", counters.totalResources, " resource/s processed)")
 	log.Print("[INFO] In ", counters.taggedFiles, " file/s (out of ", counters.totalFiles, " file/s processed)")
 
 	return nil
 }
 
-func tagDirectoryResources(dir string, filter string, matches []string, tags string, isSkipTerratagFiles bool, tfVersion convert.Version, rename bool) counters {
+func tagDirectoryResources(dir string, filter string, matches []string, tags string, isSkipTerratagFiles bool, tfVersion convert.Version, rename bool, terragrunt bool) counters {
```

Comment:
Can we move away to an `args` struct/object and not use so many parameters at once? 🙏

Reply:
Sure!

Reply:
refactored.